get_github_coderabbit_review/test_github_api_refactor.py

```
@@ -0,0 +1,182 @@
+def test_post_comment_api():
+    """Test the new REST API-based comment posting."""
+    print("🧪 Testing GitHub API comment posting...")
+
+    client = GitHubClient()
+
+    # Check authentication first
+    try:
+        if not client.is_authenticated():
+            print("❌ GitHub CLI is not authenticated. Please run 'gh auth login'")
+            return False
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)